weave/trace_server/external_to_internal_trace_server_adapter.py

```diff
@@ -226,7 +226,7 @@ def ops_query(self, req: tsi.OpQueryReq) -> tsi.OpQueryRes:
         original_project_id = req.project_id
         req.project_id = self._idc.ext_to_int_project_id(original_project_id)
         res = self._ref_apply(self._internal_trace_server.ops_query, req)
-        for op in res.ops:
+        for op in res.op_objs:
```

Comment:
I was getting type errors here, but i dont understand how this is working...

Reply:
nice

we don't implement these on server so no one every hit them